Modificación de la imagen del detall del artículo

raquelrh opened the pull request:
Se modifica la imagen del resumen asociada al detalle del artículo

raquelrh committed:
Modificación de la imagen del detall del artículo

Se modifica la imagen del resumen asociada al detalle del artículo

hectorri reviewed (changes requested):
La foto no se corresponde con la Agente Romanoff interpretada por Scarlett Johanson.

https://vignette.wikia.nocookie.net/ironman/images/2/26/Photo%28850%29.jpg

raquelrh committed:
Modificación de imagen

Se modifica la foto para que se corresponde con la Agente Romanoff interpretada por Scarlett Johanson.

raquelrh commented:
Se han modificado las imágenes a partir del enlace

hectorri reviewed (approved):
Cambios aprobados
